conf/application.conf

```
@@ -106,7 +106,7 @@ play {
 }
 
 # Set to false will close the CFP for speakers
-cfp.isOpen=${CFP_IS_OPEN}
+cfp.isOpen=true
```

```suggestion
cfp.isOpen=${CFP_IS_OPEN}
```
It is important to keep it configurable from an environment variable, it is how @nicmarti changes it in the production env

---

@dwursteisen Note that docker is loading the file run.env and you can customize it